### ISSUE TITLE
general question | can we specify the port with {port} when using -x

### ISSUE THREAD

**Issue by matth-c3** [wouterdebie/i2cssh#119]
Hello, 

Is it possible to specify the port when using `-x` ? it's necessary to specify the port for postgres client or cassandra ( `-p <port>` ).

Thanks

**Comment by wouterdebie** [comment]
`-x` pretty much disregards any ssh parameters and you can construct a command however you want. So let's say that you the following config:

```
version: 2
clusters:
  mycluster:
    - host1
    - host2
```

and you run `i2cssh mycluster -x "myclient --some-param --another-param -p 8000 -h {host}"`, i2cssh will execute:

`myclient --some-param --another-param -p 8000 -h host1` and `myclient --some-param --another-param -p 8000 -h host2` in different terminal windows.

In other words, you can pass anything to `-x` and `{host}` will be substituted with whatever you want.

**Comment by matth-c3** [comment]
Hello @wouterdebie,

make sense, I will make aliases command and connect accordingly using `-x` .

Thank you, your tool make my life easier every day.

**Comment by wouterdebie** [comment]
Maybe a quick tip: you can configure `custom_command` in your `~/.i2csshrc` as well to always override the command:

```
version: 2
clusters:
  my_ssh_cluster:
    hosts:
      - host1
      - host2
  my_pg_cluster:
    custom_command: pgsql -h {host} -p 5432
    hosts:
      - host1
      - host2
```

This way you can use `i2cssh my_pg_cluster` to connect to your cluster using `psql` instead of `ssh`.
